### ISSUE TITLE
Exclude similar looking characters

### ISSUE THREAD

**Issue by karentamrazyan** [freewarelovers/PasswordGenerator#1]
Some characters are looking very similar to the other one, so that it's sometimes difficult to distinguish them.
A great example is the capital letter "O", which looks very similar to the character/number "0" (zero).
We want to provide a feature to avoid this kind of characters from the generated password.

Here are further characters:
- 0 (number zero)
- O (capital o)
- 1 (number one)
- l (small l)
- ...

**Comment by JKHunter315** [comment]
Why don't we make a library of characters that exclude those hard to identify ones? I dont know the details of the crypto API but if it provides an array of letters/ numbers we could splice them out

**Comment by karentamrazyan** [comment]
Good point @JKHunter315! Started this wiki page [Similar Characters](Similar-Characters) to collect all "similar" characters in one place.
